tests/test_runtime.py

```diff
@@ -256,30 +250,12 @@ class TestCustomVQE:
     def test_hamiltonian_to_list_string(self):
         """Test the function that transforms a PennyLane Hamiltonian to a list string Hamiltonian."""
         coeffs = [1, 1]
-        obs = [qml.PauliX(0) @ qml.PauliX(2), qml.Hadamard(0) @ qml.PauliZ(1)]
+        obs = [qml.PauliX(0) @ qml.PauliX(2), qml.PauliY(0) @ qml.PauliZ(1)]
```

Comment:
Was there an issue with Hadamard gate?

Reply:
We used to [manually convert Hamiltonian terms into a QuantumCircuit](https://github.com/PennyLaneAI/pennylane-qiskit/blob/bf37d43293d11c82a28af06f7a8b7bd5c643be2a/pennylane_qiskit/vqe_runtime_program.py#L28). The IBMQ-provided [VQE program](https://quantum-computing.ibm.com/services/programs/prototypes?program=vqe) states that the `operator` parameter is "The Hamiltonian whose smallest eigenvalue we're trying to find. Should be PauliSumOp". So I'm converting PennyLane Hamiltonians to Qiskit PauliSumOps. Hadamard can't be in a PauliSumOp. I _could_ re-express it in the Pauli basis, but (at least, for this PR) I'd rather just fail and advise the user to use Pauli operators only.

Reply:
Ok thanks! It works